docs/organizations/managing-security-and-risk.md

```diff
@@ -201,21 +217,58 @@ The following table describes how finding statuses map to deadlines:
     </tbody>
 </table>
 
-## Finding severities and deadlines {: id="item-severities-and-deadlines"}
+## Supported security categories
```

Comment:
Note: Feel free to ignore this section when reviewing. I moved it to ensure consistency with the UI.